website/docs/docs/dbt-cloud-apis/access-metadata-api.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Access the Metadata API"
+id: "access-metadata-api"
+---
+
+Use the Metadata API to evaluate data health in the long-term or at a moment-in-time. This section will explain how to access and query the Metadata API and what you'll need.
+
+<Snippet src="metadata-api-prerequisites" />
+
+We provide a [GraphQL explorer](https://metadata.cloud.getdbt.com/graphql) for this API where you can run ad-hoc queries or browse the schema. As GraphQL provides a self-describing API, the schema shown in the GraphiQL interface is an accurate representation of the graph and fields available to query. 
+
+1. Create a [service account token](/docs/dbt-cloud-apis/service-tokens) to authorize requests. dbt Cloud Admin users can generate a _Metadata Only_ service token, which can be used to execute a specific query against the Metadata API for authorization of requests.
+
+2. Access the API by using the endpoint `https://metadata.{YOUR_ACCESS_URL}/graphql`. 
+    * Replace `{YOUR_ACCESS_URL}` with the appropriate [Access URL](/docs/deploy/regions-ip-addresses) for your region and plan. For example, if your multi-tenant region is North America, your endpoint is `https://metadata.cloud.getdbt.com/graphql`. If your multi-tenant region is EMEA, your endpoint is `https://metadata.emea.dbt.com/graphql`.
+
+3. Go to the [GraphQL API](https://metadata.cloud.getdbt.com/graphql) sandbox, which supports arbitrary queries. 
+
+4. To begin querying the Metadata API, refer to the [query documentation](/docs/dbt-cloud-apis/metadata-querying.md) for details on how to use GraphiQL.
```

Comment:
Should we explain the difference between GraphQL and GraphiQL somewhere in this doc, for the folks who are new to this?

Reply:
yep good shout!

Reply:
i've reworked this section and moved some content to the top - what do you think @matthewshaver ?

Reply:
We should just say GraphQL. GraphiQL just refers to that IDE